src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx

```diff
@@ -24,7 +37,7 @@ export const VanguardSeriesWrapper: React.SFC<{
         // prevents overlapping nav on jump-link
       />
       <Box height="95vh" mb={80}>
-        <VanguardVideoBackground article={article} />
+        {isVideo && <IntroSVGVideo id={getVideoID(slugifiedTitle)} url={url} />}
         <Box mx="auto" maxWidth={1400} px={4}>
```

Comment:
Based on the screenshots it looks like we may need to change the `z-index` on these titles, which appear to be behind the videos.

Reply:
Yeah, the video comp itself as a `z-index` of `-1` so it _should_ be rendering behind the text but the header text and the svgs have a `position:absolute` prop which may be why this appears to be happening. Anyways I'll address in the next PR which is to get the text-inversions and transforms working!

Reply:
We probably need to wrap the parent container in `position: relative`

Reply:
Happy to pair on this if you want